### PR TITLE
[appveyor] fix ini matrix race conditions

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

**Comment by stof** [comment]
So, no more trick to run only the skipped tests with ini-max ?

**Comment by nicolas-grekas** [comment]
The skipped persistence is still there, triggered by the SYMFONY_PHPUNIT_SKIPPED_TESTS env var.
But appveyor kills the build on the first failing line, which is annoying. I'm going to fix that now

File: phpunit [hunk 1]
```diff
@@ -67,7 +67,7 @@ EOPHP
 }
 
 $cmd = array_map('Symfony\Component\Process\ProcessUtils::escapeArgument', $argv);
-$exit = 0;
+$exit = 1;
```
Comment (stof) [inline phpunit]: this looks wrong to me

Comment (nicolas-grekas) [inline phpunit]: to be removed, I'm just testing how things works on appveyor :)

File: phpunit [hunk 2]
```diff
@@ -124,6 +112,7 @@ if (isset($argv[1]) && 'symfony' === $argv[1]) {
                 $exit = 1;
                 echo "\033[41mKO\033[0m $component\n\n";
             }
+            break;
```
Comment (nicolas-grekas) [inline phpunit]: to be removed

**Comment by nicolas-grekas** [comment]
Ready, should also reduce test transiency

**Pull request merged**